pkg/innerring/innerring.go

```diff
@@ -741,7 +741,7 @@ func New(ctx context.Context, log *logger.Logger, cfg *viper.Viper, errChan chan
 	containerProcessor, err := container.New(&container.Params{
 		Log:             log,
 		PoolSize:        cfg.GetInt("workers.container"),
-		AlphabetState:   server,
```

Comment:
Won't we confuse it with storage node state?

Reply:
IMO no since `NodeState` implies the current local node. We often refer to storage nodes as just nodes, but out of context this is too general.